### ISSUE TITLE
Error when adblocker is active.

### ISSUE THREAD

**Issue by Blackcatz1911** [AmazingDreams/vue-matomo#6]
```
Failed to load resource: net::ERR_BLOCKED_BY_CLIENT
vue-matomo.js:31 Error loading script Event
(anonymous) @ vue-matomo.js:31
:1 Uncaught (in promise) Event
```
The uncaught error should be caught and handled as exception.

**Comment by AmazingDreams** [comment]
This exception is already being caught, the original exception is logged. I'll add a more extensive message, it's currently only 'Error loading script'

**Comment by AmazingDreams** [comment]
This is fixed in release v0.1.10
